Ports/.port_include.sh

```diff
@@ -77,9 +80,34 @@ run() {
     echo "+ $@"
     (cd "$workdir" && "$@")
 }
-run_replace_in_file(){
+run_replace_in_file() {
     run perl -p -i -e "$1" $2
 }
+install_launcher() {
+    if [ -z "$launcher_name" ] || [ -z "${launcher_category}" ] || [ -z "${launcher_command}" ]; then
+        return
+    fi
+    script_name="${launcher_name,,}"
+    script_name="${script_name// /}"
+    mkdir -p $DESTDIR/usr/local/libexec
+    cat >$DESTDIR/usr/local/libexec/$script_name <<SCRIPT
+#!/bin/sh
+set -e
+cd "\$(dirname -- "\$(which -- $(printf %q "${launcher_command%% *}"))")"
```

Comment:
https://github.com/SerenityOS/serenity/pull/6544#discussion_r618168564

You missed this messages, you need a `--` after `cd`.

Reply:
Why did you remove it though? did it give you errors when you tried adding it?

Reply:
Aye, `cd` doesn't seem to support `--` yet.

Reply:
weird, I remember testing it on serenity yesterday.

I also tested it again right now and our `sh` seems to implement it.
![flie](https://user-images.githubusercontent.com/20175435/115836283-953df200-a417-11eb-9292-141a2da2916e.png)
![file2](https://user-images.githubusercontent.com/20175435/115836982-670ce200-a418-11eb-9dee-c486c69b383a.png)

----------

If you are just saying serenity doesn't seem to support it because it is not listed in the help menu, `--` is automatically supported by anything that uses an `ArgsParser` even that `ArgsParser` doesn't have any options.

Reply:
Ok, I guess it's a bit more complicated than just "doesn't seem to be supported":

![Screenshot from 2021-04-23 10-14-05](https://user-images.githubusercontent.com/388571/115841087-b81ed500-a41c-11eb-8933-635111b1a8a6.png)

(`x` doesn't  exist, so that error is OK.)

Reply:
Alright then, that's fixed. :partying_face: